users/models.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+from enum import Enum
+
+
+class UserType(Enum):
+    STUDENT = "STUDENT"
+    LECTURER = "LECTURER"
+    GUEST = "GUEST"
+
+
+class UsersTypes(models.Model):
+    name = models.CharField(max_length=8,
+                            choices=[(tag, tag.value) for tag in UserType])
```

Comment:
@proszkie dlaczego tutaj jest to jako Tuple?

Reply:
Zmienione

Reply:
https://docs.djangoproject.com/en/3.1/ref/models/fields/

Tak to działa w przypadku enum field, ale po konsultacji z @PKramek trochę to uprościliśmy ;)